meson-scripts/install_rust_user_scheds

```diff
@@ -15,13 +15,41 @@ for manifest in "$MESON_SOURCE_ROOT"/scheds/rust/*/Cargo.toml; do
 
     bins=($(ls -t "${target_dir}/"*"/${name}"))
     if [ ${#bins[@]} -lt 1 ]; then
-	echo "Cannot find a binary for $name under $target_dir" 1>&2
-	exit 1
+        echo "Cannot find a binary for $name under $target_dir" 1>&2
+        exit 1
     fi
 
     # XXX - Can we detect the current buildtype and install the correct one?
     if [ ${#bins[@]} -gt 1 ]; then
-	echo "Found multiple binaries for $name under $target_dir, installing the latest which may not be what you want" 1>&2
+        echo "Found multiple binaries for $name under $target_dir, installing the latest which may not be what you want" 1>&2
+    fi
+
+    install -D "${bins[0]}" "${DESTDIR}/${MESON_INSTALL_PREFIX}/bin/${name}"
+done
+
+for manifest in "$MESON_SOURCE_ROOT"/rust/*/Cargo.toml; do
+    source_dir="${manifest%/Cargo.toml}"
+    target_dir="${MESON_BUILD_ROOT}/rust"
+    name="${source_dir##*/}"
+
+    if [ ! -f "${target_dir}/"*"/${name}" ]; then
+        continue
+    fi
+
+    bins=($(ls -t "${target_dir}/"*"/${name}"))
+    if [ ${#bins[@]} -lt 1 ]; then
+        echo "Skipping. Cannot find a binary for $name under $target_dir" 1>&2
+        continue
+    fi
+
+    # XXX - Can we detect the current buildtype and install the correct one?
+    if [ ${#bins[@]} -gt 1 ]; then
+        echo "Found multiple binaries for $name under $target_dir, installing the latest which may not be what you want" 1>&2
+    fi
+
+    # Install bus spec for scx_loader
+    if [ "$name" = "scx_loader" ]; then
+        install -D -m 0644 "${source_dir}/org.scx.Loader.conf" "${DESTDIR}/usr/share/dbus-1/system.d/org.scx.Loader.conf"
```

Comment:
I wonder whether the right thing to do is invoking `cargo install` with the right parameters. e.g. if someone does `cargo install scx_loader`, it should also install the dbus-1 file too, right?

Reply:
yeah it's required. hmm I don't think its possible to install custom files with `cargo install`

Reply:
Looks like not possible yet to install custom files with `cargo install`, but possible with tools like `cargo xtask`

https://github.com/rust-lang/cargo/issues/2729

Reply:
I see. Let's leave as-is for the time being.